.env.example

```diff
@@ -0,0 +1,8 @@
+# If running docker locally, set this to http://localhost:3000/api
+REACT_APP_BASE_URL=
+
+# If running docker locally, set this to ff:changeme
+USERS=
+
+# If running docker locally, this needs to be set
+# GOOGLE_APPLICATION_CREDENTIALS=
```

Comment:
This bit isn’t quite set up to work when running docker locally, but I think it’s quite close. I don’t really want to fix that in this PR.